packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -495,4 +514,19 @@ export enum LaunchType {
    * The service will be launched using the FARGATE launch type
    */
   FARGATE = 'FARGATE'
+}
+
+/**
+ * Propagate tags from either services or task definitions
```

Comment:
nit: use singular of service and task definition

Reply:
done!